client/gutenberg/extensions/tiled-gallery/gallery-grid.jsx

```diff
@@ -24,32 +24,36 @@ class TiledGalleryGrid extends Component {
 
 	pendingRaf = null;
 
-	handleResize = entries => {
-		if ( this.pendingRaf ) {
-			cancelAnimationFrame( this.pendingRaf );
-		}
-		this.pendingRaf = requestAnimationFrame( () => {
-			this.pendingRaf = null;
-			for ( const entry of entries ) {
-				const { width } = entry.contentRect;
-				if ( width && width !== this.state.width ) {
-					this.setWidth( width );
+	handleResize = this.props.noResize
+		? () => {}
```

Comment:
I take that this ternary is just an optimization because we already have `if ( ! this.props.noResize )` in `componentDidMount` ?

Reply:
At this point, it's redundant as you note. I expect we may start to call resize on mount to get initial sizes set up. This is also complete, `noResize` really will not resize.